controls/radcoloreditor/features/color-mode.md

```diff
@@ -14,15 +14,35 @@ The __RadColorEditor__ control allows supports the most popular color models. Yo
 
 The __ColorMode__ property is an enumeration of type __ColorMode__ and it exposed the following members:				
 
+#### __[XAML] Example 1: Set the color model to RGB__
+{{region radcoloreditor-features-color-mode_0}}
+	<telerik:RadColorEditor x:Name="colorEditor" SelectedColor="Red" ColorMode="RGB"/>
+{{endregion}}
+
 * __RGB__ - sets the color model to[RGB(Red, Green and Blue)](http://en.wikipedia.org/wiki/RGB_color_model)  

```

Comment:
The convention for list items with title is to use the title with colon after it instead of a dash. Also, the existence after the title part starts with a capital. For example:

* __RGB__: Sets the color model...